author.rb

```diff
@@ -0,0 +1,10 @@
+class Author
+  attr_accessor :id, :first_name, :last_name, :items
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items []
```

Comment:
The instance variable for items isn't set to the empty array yet. I suggest you add '=' to fix this oversight.

Reply:
Okay, thank you. I will sort it out now